setup.py

```diff
@@ -85,9 +85,6 @@ def run(self):
     "drf-spectacular",
     # pulp-container 2.6 requires pulpcore >=3.12.1
     "pulp-container>=2.7.0,<2.8.0",
-    # click 8 requires py38,
-    # can be removed once we require >=py38
-    "click==7.1.2",
```

Comment:
The requirements files still have this requirement. I recommend not running `make requirements` which updates all to latest via pip, but running these which updates just based on setup.py (i will create a make endpoint to make this eaiser)
```
pip-compile --output-file=requirements/requirements.common.txt setup.py
pip-compile --output-file=requirements/requirements.insights.txt requirements/requirements.insights.in setup.py
pip-compile --output-file=requirements/requirements.standalone.txt requirements/requirements.standalone.in setup.py
```